__tests__/app.test.js

```diff
@@ -20,4 +22,19 @@ describe('api/topics', () => {
             })
         })
     })
-})    
\ No newline at end of file
+}) 
+describe('api/', () => {
+    test('GET:200, An object describing all the available endpoints on your API', () => {
+        let endpointsObj
+        return fs.readFile(path.join(__dirname, '../endpoints.json'), 'utf-8')
```

Comment:
Not feeling super confident in this test, just purely because it's so similar to your implementation; if something about your ```readFile``` method is incorrectly formatted and doesn't work both times, how can you be sure you're getting back what you want if you're comparing them against each other? 

I think it's okay in this instance as you're just reading the file, but for other tests try to keep this in mind and consider other ways you can check the shape of the data 🙂